client/src/components/Login.tsx

```diff
@@ -420,7 +421,12 @@ const LoginWithoutI18n = (props: LoginProps) => {
             labelText={i18n._(t`Email address`)}
           />
           <div className="submit-button-group">
-            <Button variant="primary" size="small" labelText={i18n._(t`Get updates`)} />
+            <Button
```

Comment:
I forgot to ask you about this when I first added the component library buttons here, but for some reason only with the buttons within these overview cards the `cursor: pointer` is not working properly. Specifically for the `jfcl-button__label` element. Even though when inspecting it shows that it is inheriting `pointer` from the button, and nothing else should be affecting it. You have any idea what might be causing this? 
![image](https://github.com/JustFixNYC/who-owns-what/assets/16906516/75942769-26b3-4173-8e6b-7488378ac7f6) 
![image](https://github.com/JustFixNYC/who-owns-what/assets/16906516/0541810e-1554-4a33-9834-af5f3c41e6e6)

I'll just make a ticket for this and we can address later https://app.shortcut.com/justfixnyc/story/14192

Reply:
i think it's because of the way I overrode the datatable mixin in the first few rows of Card.scss. It was an easy fix so it will be in the next commit